src/Aspire.Hosting.Python/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: Experimental("ASPIREHOSTINGPYTHON001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
```

Comment:
How do we get `https://learn.microsoft.com/en-us/dotnet/aspire/diagnostics/overview#ASPIREHOSTINGPYTHON001` to be a valid URL?

cc @IEvangelist

Reply:
The `ASPIREHOSTINGPYTHON001` has to either be a heading, or you can use an inline named HTML element:

```markdown
<span name="ASPIREHOSTINGPYTHON001"></span>
```

Then the bookmark will work.

Reply:
@IEvangelist when I create the doc I use a heading, same as the other sections, but the generated id is lower-cased so the current link doesn't work. I will add the <span> this way there is no change to do in aspire, but it's unfortunate. Maybe the doc generator could also inject a name with the actual case of the heading (though that would leak everywhere).

Reply:
Yeah, that's a `dotnet/docfx` concern more than anything else. Bookmarks in URLs just look better in lowercase in my opinion, but knowing that they're always in lowercase, why not just have this code ensure that they're lowercase?

Reply:
I agree, had I known I would have done it before. But I don't want to block release/9.0 insertion on that. Will do it in main once 9.0 has shipped and is merged back.